codegen/golang.go

```diff
@@ -87,6 +87,20 @@ func WithDescriptionsEnabled() TableOptions {
 	}
 }
 
+// Unwrap specific struct fields (1 level deep only)
+func WithUnwrapFieldsStructs(fields []string) TableOptions {
+	return func(t *TableDefinition) {
+		t.structFieldsToUnwrap = fields
+	}
+}
+
+// Unwrap all fields that are embedded structs (1 level deep only)
+func WithUnwrapAllEmbeddedStructs() TableOptions {
```

Comment:
Handles the use case of:
```
type testStructWithNonEmbeddedStruct struct {
  *Properties
}
```

Without the need to configure each struct we'd like to unwrap